xml/selinux.xml

```diff
@@ -88,10 +88,8 @@
   </para>-->
 
   <para>
-   &selnx; uses labels attached to objects (for example, files, binaries,
-   and network sockets) and uses them to determine privilege boundaries, 
-   thereby building up a level of confinement that can span more than a 
-   process, or even the whole system.
+   &selnx; uses labels attached to objects (for example, files
+   and network sockets) and uses them for access decision.
```

Comment:
Not sure about "access decision", it seems a bit vague. How about "access control decisions" ?